packages/Webkul/ElasticSearch/src/Console/Command/Indexer.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+namespace Webkul\ElasticSearch\Console\Command;
+
+use Illuminate\Console\Command;
+use Webkul\Attribute\Models\Attribute;
+use Webkul\Attribute\Models\AttributeFamily;
+use Webkul\Category\Models\Category;
+use Webkul\Category\Models\CategoryField;
+use Webkul\Core\Facades\ElasticSearch;
+use Webkul\Core\Models\Channel;
+use Webkul\Core\Models\Locale;
+use Webkul\Product\Models\Product;
+
+class Indexer extends Command
+{
+    protected $signature = 'indexer:index';
+
+    protected $description = 'Index all into Elasticsearch';
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        if (env('ELASTICSEARCH_ENABLED', false)) {
+            $start = microtime(true);
+
+            // Indexing Products
+            $products = Product::all();
+
+            $productIndex = strtolower('products');
+
+            $dbProductIds = $products->pluck('id')->toArray();
+
+            $elasticProductIds = collect(Elasticsearch::search([
+                'index' => $productIndex,
+                'body'  => [
+                    '_source' => false, // Fetch only IDs
+                    'query'   => [
+                        'match_all' => new \stdClass,
+                    ],
+                ],
+            ])['hits']['hits'])->pluck('_id')->map(fn ($id) => (int) $id)->toArray();
+
+            $productsToDelete = array_diff($elasticProductIds, $dbProductIds);
```

Comment:
Create seperate indexers for each entity or have options so only one of them can also be run like if i want to reindex only data for the categories it can be reindexed without all other data

Reply:
Created separate indexers for Products and Categories.